circleguard/gui.py

```diff
@@ -353,12 +731,72 @@ def __init__(self):
         self.run_button.setText("Run")
         self.run_button.clicked.connect(self.add_circleguard_run)
 
-        layout = QVBoxLayout()
-        layout.addWidget(tabs)
-        layout.addWidget(self.terminal)
-        layout.addWidget(self.run_button)
+        layout = QGridLayout()
+        layout.addWidget(self.loadables_combobox, 0, 0, 1, 7)
+        layout.addWidget(self.loadables_button, 0, 7, 1, 1)
+        layout.addWidget(self.checks_combobox, 0, 8, 1, 7)
+        layout.addWidget(self.checks_button, 0, 15, 1, 1)
+        layout.addWidget(self.loadables_scrollarea, 1, 0, 4, 8)
+        layout.addWidget(self.checks_scrollarea, 1, 8, 4, 8)
+        layout.addWidget(self.terminal, 5, 0, 2, 16)
+        layout.addWidget(self.run_button, 7, 0, 1, 16)
+
         self.setLayout(layout)
-        self.switch_run_button()  # disable run button if there is no api key
+        self.check_run_button() # disable run button if there is no api key
+
+    # am well aware that there's much duplicated code between remove_loadable,
+    # remove_check, add_loadable, and add_check. Don't feel like writing
+    # more generic functions for them right now.
```

Comment:
unnecessary comment

Reply:
disagree, it's a reminder to clean up the code later